tests/data/placebo/test_message_broker_vpc_default_filter/ec2.DescribeVpcs_1.json

```diff
@@ -0,0 +1,184 @@
+{
```

Comment:
Do you need the data from all 6 VPCs here?

For the purposes of keeping these placebo data files small, it might be nice to cut out extraneous VPC data. Maybe keep 1-2 that provide the security groups and message brokers that feed into your unit test's match, and cut down on the number of extra ones that don't match your filter.